src/com/palmergames/bukkit/towny/command/InviteCommand.java

```diff
@@ -148,22 +147,25 @@ public static void parseDeny(Player player, String[] args) {
 				return;
 			}
 		}
-		ListMultimap<Town, Resident> town2residents = InviteHandler.getTowntoresidentinvites();
-		if (town2residents.containsKey(town)) {
-			if (town2residents.get(town).contains(resident)) {
-				for (Invite invite : resident.getReceivedInvites()) {
-					if (invite.getSender().equals(town)) {
-						try {
-							InviteHandler.declineInvite(invite, false);
-							return;
-						} catch (InvalidObjectException e) {
-							e.printStackTrace(); // Shouldn't happen, however like i said a fallback
-						}
-					}
+		
+		Invite toDecline = null;
+
+		for (Invite invite : InviteHandler.getActiveInvites()) {
+			if (invite.getSender().equals(town)) {
```

Comment:
These if statements should be merged into one imo, there are a few cases of them throughout this diff.

Reply:
Ah github doesnt show anything below the line I comment, anyway, this line and the line below create extra indentation and this is the case in several places.

Reply:
I've collapsed all the if statements in the latest commit